coffee-mode.el

```diff
@@ -529,6 +529,14 @@ output in a compilation buffer."
            (generate-new-buffer-name coffee-compiled-buffer-name))))
     (compile (concat coffee-command " " args))))
 
+(defun coffee-toggle-fatness ()
+  "Toggle fatness of a coffee function arrow."
+  (interactive)
+  (save-excursion
+    (re-search-backward "[-=]>" )
```

Comment:
Exception is raised if there is no `->` or `=>` before current position. Please set `no-error` option.